organization/src/integrationTest/kotlin/com/cosmotech/organization/service/OrganizationServiceIntegrationTest.kt

```diff
@@ -1283,1447 +1283,1502 @@ class OrganizationServiceIntegrationTest : CsmRedisTestBase() {
-          }
-          val otherUserACL = OrganizationAccessControl(id = TEST_USER_ID, role = ROLE_VIEWER)
-          assertNotNull(
-              organizationApiService.addOrganizationAccessControl(
-                  organizationRegistered.id!!, otherUserACL))
-
-          val otherUserACLRetrieved =
-              organizationApiService.getOrganizationAccessControl(
-                  organizationRegistered.id!!, TEST_USER_ID)
-          assertEquals(otherUserACL, otherUserACLRetrieved)
-        }
-      }
-
-      @Test
-      fun `updateOrganizationAccessControl with owned organization cannot update last admin`() {
-        assertThrows<CsmAccessForbiddenException> {
-          val name = "o-connector-test-1"
-          val organizationRegistered =
-              organizationApiService.registerOrganization(createTestOrganization(name))
-          organizationApiService.updateOrganizationAccessControl(
-              organizationRegistered.id!!, TEST_ADMIN_USER_ID, OrganizationRole(role = ROLE_VIEWER))
-        }
-      }
-
-      @Test
-      fun `updateOrganizationAccessControl with owned organization can update user (!= ROLE_NONE)`() {
-        assertDoesNotThrow {
-          val name = "o-connector-test-1"
-          val organizationRegistered =
-              organizationApiService.registerOrganization(createTestOrganization(name))
-
-          organizationApiService.addOrganizationAccessControl(
-              organizationRegistered.id!!,
-              OrganizationAccessControl(id = OTHER_TEST_USER_ID, role = ROLE_VIEWER))
-
-          assertNotNull(
-              organizationApiService.updateOrganizationAccessControl(
-                  organizationRegistered.id!!,
-                  OTHER_TEST_USER_ID,
-                  OrganizationRole(role = ROLE_EDITOR)))
-
-          val userACLRetrieved =
-              organizationApiService.getOrganizationAccessControl(
-                  organizationRegistered.id!!, OTHER_TEST_USER_ID)
-          assertNotEquals(ROLE_VIEWER, userACLRetrieved.role)
-          assertEquals(ROLE_EDITOR, userACLRetrieved.role)
-        }
-      }
-
-      @Test
-      fun `updateOrganizationAccessControl with owned organization cannot update user (= ROLE_NONE)`() {
-        assertThrows<CsmClientException> {
-          val name = "o-connector-test-1"
-          val organizationRegistered =
-              organizationApiService.registerOrganization(createTestOrganization(name))
-
-          organizationApiService.updateOrganizationAccessControl(
-              organizationRegistered.id!!, TEST_USER_ID, OrganizationRole(role = ROLE_NONE))
-        }
+        assertEquals(otherUserACL, otherUserACLRetrieved)
       }
+    }
 
-      @Test
-      fun `updateOrganizationAccessControl with owned organization and unknown ACL user`() {
+    @Test
+    fun `addOrganizationAccessControl with not owned organization and no PERMISSION_WRITE_SECURITY`() {
+      assertDoesNotThrow {
```

Comment:
assertDoesNotThrow should be specific else it's the same as not having it

Reply:
same comment as above